src/lib/components/inventory/selected_item_info.ts

```diff
@@ -222,5 +226,11 @@ export class SelectedItemInfo extends FloatElement {
                 .fetch({steam_id64: g_ActiveInventory?.m_owner.strSteamId})
                 .then((stall) => (this.stall = stall));
         }
+
+        // Make sure the parent container can overflow
+        const parentContainer = document.querySelector<HTMLElement>('.item_desc_content');
```

Comment:
We should be finding this element relative to the current element, instead of query selecting across the whole document.

Reply:
I reworked it with `this.closest(...)` now, I assume this is what you had in mind?